docs_user/modules/con_block-storage-service-limitations.adoc

```diff
@@ -0,0 +1,20 @@
+[id="block-storage-limitations_{context}"]
+
+= Limitations for adopting the {block_storage}
+
+There are currently some limitations that are worth highlighting; some are
+related to this guideline while some to the operator:
+
+* There is no global `nodeSelector` for all {block_storage_first_ref} volumes, so it needs to be
+specified per backend.  This may change in the future.
```

Comment:
```suggestion
specified per backend. 
```

Maybe we want this but in general should lean away from future looking statements.

Reply:
Same for similar statements below